rpc_jobs/pull_request_whisperer.yml

```diff
@@ -0,0 +1,78 @@
+- project:
+    name: 'Pull-Request-Whisperer-Jobs'
+    series:
+      - all_branches:
+          branches: ".*"
+    repo:
+      - rpc_openstack:
```

Comment:
It would probably make sense to separate this into the project files.

Reply:
Sure, splitting these out.